Gopkg.toml

```diff
@@ -535,3 +535,7 @@
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
   revision = "4d05e43ad98c1edcf2f52291685bd9bbc12fd2cd"
+
+[[constraint]]
+  revision = "66506e52db051b5e238aa63ec4dad0a790fbc236"
+  name = "github.com/bcsaller/jsonschema"
```

Comment:
newline at end of file?

Reply:
damn it, i'm installing a plugin into my IDE....